docs/design/README.md

```diff
@@ -297,8 +297,7 @@ represent that value.
 
 > References:
 >
-> -   [Integer literal semantics](lexical_conventions/numeric_literals.md#integer-literals)
-> -   [Integer Literals](numeric_literals.md#integers)
+> -   [Integer literal syntax](lexical_conventions/numeric_literals.md#integer-literals)
```

Comment:
I updated this, there's now references to both my new doc and proposal #144 . Do we want both of those references, or is the #144 reference redundant since this new doc formalizes #144 anyway?

Reply:
I'd keep them all.